projects/data-structures-algorithms-sprint.md

```diff
@@ -44,25 +41,29 @@ Groups will present introductory overviews to topics in this order:
 5. Hash tables
 6. Trees
 
-#### Day 4
-You will review what you’ve learned so far and practice the exercises for each data structure. Homework for the evening: read about your assigned algorithms topics and highlight or outline the most important parts.
-
-#### Day 5
-In the morning, group members compare notes about algorithms topics, start creating introductory slides, and decide who will present what. In the afternoon, groups will plan out demos, select beginner exercises and resources, polish slides, and practice presenting their introductory topic overviews (remember that reading is discouraged and eye contact and a loud, confident voice are encouraged).
 
-#### Day 6
-Groups will present introductory overviews to topics in this order:
 
-1. Searching: Binary
-2. Searching: Depth and Breadth
+#### Day 2
+All the groups will practice and then present the Data structure topics at the end of the day:
+1. Stacks
+2. Queues
+3. Deques
+4. Linked lists
+5. Hash tables
+6. Trees
 
-#### Day 7
-Groups will present introductory overviews to topics in this order:
+#### Day 3
```

Comment:
Could you please make this into a numbered or bulleted list like the others?

Reply:
ohk sure. I will make the required  change. IF you want any other changes also you could mention.

Reply:
Change line 56 to “Prepare Algorithms”

Thanks for making the list!